spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -112,6 +114,53 @@ trait DeltaTableOperations extends AnalysisHelper { self: DeltaTable =>
     toDataset(sparkSession, restore)
   }
 
+  protected def executeClone(
+      table: DeltaTableV2,
+      target: String,
+      replace: Boolean,
+      properties: Map[String, String],
+      versionAsOf: Option[Long] = None,
+      timestampAsOf: Option[String] = None): DeltaTable = withActiveSession(sparkSession) {
+    val (targetRelation, targetPath) = if (new Path(target).isAbsolute()) {
+      (UnresolvedRelation(TableIdentifier(target, Some("delta"))), Some(target))
+    } else {
+      val tableIdentifier = sparkSession
+        .sessionState
+        .sqlParser
+        .parseTableIdentifier(target)
+      (UnresolvedRelation(tableIdentifier), None)
+    }
+
+    val sourceIdentifier = table.getTableIdentifierIfExists.map(id =>
+      Identifier.of(id.database.toArray, id.table))
+    val sourceRelation = DataSourceV2Relation.create(table, None, sourceIdentifier)
+
+    val maybeTimeTravelSource = if (timestampAsOf.isDefined || versionAsOf.isDefined) {
+      TimeTravel(
+        sourceRelation,
+        timestampAsOf.map(Literal(_)),
+        versionAsOf,
+        None
+      )
+    } else {
+      sourceRelation
+    }
+
+    val clone = CloneTableStatement(maybeTimeTravelSource, targetRelation, false, replace, true,
```

Comment:
According to the style guide "For method and class constructor invocations, use 2 space indentation for its parameters and put each in each line when the parameters don't fit in two lines." - https://github.com/databricks/scala-style-guide

```suggestion
    val clone = CloneTableStatement(
      maybeTimeTravelSource,
      targetRelation,
      ifNotExists = false,
      replace,
      isCreateCommand = true,
      tablePropertyOverrides = properties.toMap,
      targetLocation = None)
```

Reply:
It does fit on two lines?

Reply:
I think we should add the argument names, `false` or `true` is not so clear from the get go what the argument is, let's leverage the strength of Scala